packages/manager/apps/dedicated/client/app/account/contacts/service/translations/Messages_de_DE.json

```diff
@@ -36,6 +36,5 @@
   "account_contacts_service_contact_tech": "Technischer Kontakt",
   "account_contacts_service_contact_billing": "Rechnungskontakt",
   "account_contacts_service_infos": "Sie können die Administator-, technischen und Rechnungs-Accounts Ihrer Dienste ändern.",
-  "account_contacts_service_category_OVH_CLOUD_CONNECT": "OVHcloud Connect",
-  "account_contacts_service_category_ESSENTIALS": "Essentials"
```

Comment:
this translations shouldn't be removed

Reply:
I think this got in when i rebased, ill take care of it.